index.bs

```diff
@@ -1215,6 +1215,383 @@ implementation of this step is specific to the user agent.", the controlling
 user agent may send the remote-playback-termination-request message.
 
 
+Streaming Protocol {#streaming-protocol}
+========================================
+
+The section defines the use of the Open Screen Protocol for streaming
+media from one agent to another. We refer to the agent sending media
+as the *sender* and the agent receiving the media as the *receiver*.
+
+Capabilities {#streaming-capabilities}
+--------------------------------------------
+
+<!--
+  TODO:
+
+  Put in something like this:
+
+streaming-capabilities-request {
+  request
+}
+
+streaming-capabilities-response {
+  response
+  capabilities: streaming-capabilities
+}
+
+streaming-capabilities = {
+  receive-audio: [* receive-audio-capability]
+  receive-video: [* receive-video-capability]
+  receive-data: [* receive-data-capability]
+}
+
+
+receive-audio-capability = {
+  codec-name: text
+  codec-parameters: [* codec-parameter]
+}
+
+receive-video-capability = {
+  codec-name: text
+  codec-parameters: [* codec-parameter]
+  ? max-resolution: video-resolution
+  ? max-frames-per-second: float
+  ? max-pixels-per-second: float
```

Comment:
No, because you might not have a limitation on resolution, only on pixel rate and you don't care about resolution.   However, if both resolution and framerate are as limits, that does provide an implicit lower bound on pixel rate.